webrtc-stats.html

```diff
@@ -1143,9 +1143,20 @@
              boolean                       readable;
              unsigned long long            bytesSent;
              unsigned long long            bytesReceived;
-             double                        roundTripTime;
+             double                        averageRtt;
+             double                        currentRtt;
```

Comment:
A total of Rtt/total responses is a good suggestion. Depending on the sampling rate, an endpoint would be able to measure average interval RTTs.

How often are the requests expected to be sent?